index.html

```diff
@@ -0,0 +1,1946 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>News Week Clone</title>
+		<!-- Bootstrap CSS -->
+		<link
+			rel="stylesheet"
+			href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+			integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+			crossorigin="anonymous"
+		/>
+		<link
+			href="https://fonts.googleapis.com/css?family=Roboto+Condensed"
+			rel="stylesheet"
+		/>
+		<link
+			href="https://fonts.googleapis.com/css?family=Alfa+Slab+One"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="assets/css/style.css" />
+	</head>
+	<body>
+		<header>
+			<div class="container-fluid">
+				<div class="row header-top">
+					<div class="col-md-3 header-top-item txt-left">
+						<p>Mon, Apr 15, 2019</p>
+					</div>
+					<div class="col-md-6 header-top-item txt-center">
+						<h1>NewsWeek</h1>
+					</div>
+					<div class="col-md-3 header-top-item txt-right">
+						<p>
+							SIGN IN
+							<button>SUBSCRIBE</button>
+							<span class="menu-icon"><img alt="" src="assets/images/menu.png"></span>
+						</p>
+					</div>
+				</div>
+				<div class="row navbar-container">
```

Comment:
You can use nav tag in main navigation, i think it is better to use nav tag.

Reply:
ok